expression/util.go

```diff
@@ -30,16 +30,30 @@ import (
 
 // ExtractColumns extracts all columns from an expression.
 func ExtractColumns(expr Expression) (cols []*Column) {
+	result := make([]*Column, 0, 5)
```

Comment:
Why 5?

Reply:
See the explain below.

Reply:
I think the power of 2 is better, like 8.